test/integration/scheduler/scheduler_test.go

```diff
@@ -141,28 +140,16 @@ func DoTestUnschedulableNodes(t *testing.T, cs clientset.Interface, ns *v1.Names
 	// non-namespaced objects (Nodes).
 	defer cs.Core().Nodes().DeleteCollection(nil, metav1.ListOptions{})
 
-	goodCondition := v1.NodeCondition{
-		Type:              v1.NodeReady,
-		Status:            v1.ConditionTrue,
-		Reason:            fmt.Sprintf("schedulable condition"),
-		LastHeartbeatTime: metav1.Time{time.Now()},
-	}
-	badCondition := v1.NodeCondition{
-		Type:              v1.NodeReady,
-		Status:            v1.ConditionUnknown,
-		Reason:            fmt.Sprintf("unschedulable condition"),
-		LastHeartbeatTime: metav1.Time{time.Now()},
-	}
 	// Create a new schedulable node, since we're first going to apply
 	// the unschedulable condition and verify that pods aren't scheduled.
+	// Note that scheduler only filter out nodes with unschedulable=true spec, so no need to check node conditions anymore.
```

Comment:
the `anymore` part is confusing - not sure comments in the code should reference past versions of the code?